<launch>


    <node pkg="ros_tutorial" type="follower.py" name="follower" output="screen">
        <param name="linear_speed" value="0.4" />
        <param name="angular_speed" value="0.5"/>
        <param name="goal_angle" value="90"/>
        <param name="stop_distance" value="0.1"/>
        <param name="epsilon" value="0.001"/>
        
    </node>

    <node pkg="ros_tutorial" type="monitor.py" name="monitor"></node>

    <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_empty_world.launch">
        <arg name="x_pos" value="0.0"/>
        <arg name="y_pos" value="0.0"/>
        <arg name="z_pos" value="0.0"/>
    </include>

    <node name="rviz" pkg="rviz" type="rviz"/>

</launch>